2.6/Dockerfile.rhel7

```diff
@@ -29,7 +29,6 @@ LABEL summary="$SUMMARY" \
       com.redhat.component="${RUBY_SCL}-container" \
       name="${IMAGE_NAME}" \
       version="${RUBY_VERSION}" \
-      com.redhat.license_terms="https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI" \
```

Comment:
Please change this to `com.redhat.license_terms="https://www.redhat.com/en/about/red-hat-end-user-license-agreements#rhel" ` instead.